APIWrapper/Objects/ScheduleHourContent.cs

```diff
@@ -0,0 +1,67 @@
+using APIWrapper.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace APIWrapper.Objects
+{
+    public class ScheduleHourContent
+    {
+        /// <summary>
+        ///         Every day has 13 blocks(1-13)
+        ///         block 1 starts at 7:00
+        ///         block 13 starts at 19:00
+        /// </summary>
+        public int BlockNumber { get; internal set; }
+
+        /// <summary>
+        ///         Is block blocked?
+        /// </summary>
+        public bool IsBlocked { get; internal set; }
+
+        /// <summary>
+        ///         Type of lesson - lecture, laboratory, excersise
+        /// </summary>
+        public eLessonType LessonType { get; internal set; }
+
+        /// <summary>
+        ///         Name of the teacher
+        /// </summary>
+        public string TeacherName { get; internal set; }
+
+        /// <summary>
+        ///         Room number where is lesson
+        /// </summary>
+        public string RoomName { get; internal set; }
+
+        /// <summary>
+        ///         Subject shortcut
+        /// </summary>
+        public string SubjectShortcut { get; internal set; }
+
+        /// <summary>
+        ///         Full name of subject
+        /// </summary>
+        public string SubjectName { get; internal set; }
+
+        /// <summary>
+        ///         All study groups which have lesson in the same time
+        /// </summary>
+        public List<string> StudyGroups { get; internal set; }
+
+        /// <summary>
+        ///         Type of subject - compulsory, optional, compulsoryElective
+        /// </summary>
+        public eSubjectType SubjectType { get; internal set; }
+
+        /// <summary>
+        ///         Block is empty (free block)
+        /// </summary>
+        public bool IsEmptyBlock { get; internal set; }
```

Comment:
We should send over wire only data that are necessary for proper timetable rendering. I do not think that it is a good idea to send empty blocks, it just adds more overhead without any useful information...

Reply:
I think that we will need it in the future - to know if block is free

Reply:
We will know it indirectly

Reply:
ok